neurodamus/target_manager.py

```diff
@@ -225,7 +225,7 @@ def init_hoc_manager(self, cell_manager):
         # give a TargetManager the TargetParser's completed targetList
         self.hoc = Nd.TargetManager(self.parser.targetList, cell_manager)
 
-    def get_target_points(self, target, cell_manager, cell_use_compartment_cast, **kw):
+    def get_target_points(self, target, cell_manager, **kw):
```

Comment:
Is `cell_use_compartment_case` not needed any more? If so, we can remove it from the docstring and in `node.py`